Source/ProgressionSystemRuntime/Public/Data/PSWorldSubsystem.h

```diff
@@ -100,9 +117,18 @@ class PROGRESSIONSYSTEMRUNTIME_API UPSWorldSubsystem : public UWorldSubsystem
 	UPROPERTY(VisibleInstanceOnly, BlueprintReadWrite, Transient, Category = "C++", meta = (BlueprintProtected, DisplayName = "Progression System Spot Component"))
 	TObjectPtr<class UPSSpotComponent> PSCurrentSpotComponentInternal = nullptr;
 
-	/** Store the current save game instance */
+	/** Store the current save game instance
+	 * Contains the FPSSaveToDiskData which has actual data from save file */
+	UPROPERTY(VisibleInstanceOnly, BlueprintReadWrite, Category = "C++", meta = (BlueprintProtected, DisplayName = "Save Game Data Internal"))
+	TObjectPtr<class UPSSaveGameData> SaveGameDataInternal = nullptr;
+
+	/** Store the Progression Settings data instance */
 	UPROPERTY(VisibleInstanceOnly, BlueprintReadWrite, Category = "C++", meta = (BlueprintProtected, DisplayName = "Save Game Instance"))
-	TObjectPtr<class UPSSaveGameData> SaveGameInstanceInternal = nullptr;
+	TMap<FName, FPSRowData> ProgressionSettingsDataInternal;
```

Comment:
- add `Transient` specifier for this runtime array
- please update comment to explain approach more precisely like it store default values from data table that are cached once on starting the game and never change later

Reply:
done: https://github.com/h4rdmol/ProgressionSystem/pull/18/commits/d6e9d76b00d2682fa814e723733a707054b00183